tests/functional/builtins/codegen/test_create_functions.py

```diff
@@ -95,19 +96,17 @@ def test2(a: uint256) -> Bytes[100]:
     c = get_contract(code)
 
     assert c.hello(1) == b"hello world!"
-    c.test(transact={})
+    c.test()
     assert c.test2(1) == b"hello world!"
 
-    with tx_failed():
+    with tx_failed(exc_text="invaliddddd"):
         c.test2(0)
 
-    GAS_SENT = 30000
-    tx_hash = c.test2(0, transact={"gas": GAS_SENT})
-
-    receipt = w3.eth.get_transaction_receipt(tx_hash)
+    gas_sent = 30000
+    with tx_failed(exc_text="invaliddddd"):
+        c.test2(0, gas=gas_sent)
 
-    assert receipt["status"] == 0
-    assert receipt["gasUsed"] < GAS_SENT
+    assert env.last_result.gas_used < gas_sent
```

Comment:
we should add a comment why this is here (follow up for https://github.com/vyperlang/vyper/pull/3846#discussion_r1557511851)

Reply:
I don't know why this was here :cry: